pubspec.yaml

```diff
@@ -8,19 +8,19 @@ homepage: https://www.github.com/dart-lang/reflectable
 environment:
   sdk: '>=1.12.0 <3.0.0'
 dependencies:
-  analyzer: '>=0.33.0 <0.35.0'
-  build: ^1.0.0
-  build_resolvers: '>=0.2.0 <2.0.0'
-  build_config: ^0.3.0
-  build_runner: ^1.0.0
-  build_runner_core: '>=1.0.0 <3.0.0'
-  dart_style: '>=0.2.0 <2.0.0'
+  analyzer: '>=0.36.0 <0.38.0'
+  build: ^1.1.0
+  build_resolvers: '>=1.0.0 <2.0.0'
+  build_config: ^0.4.0
+  build_runner: ^1.6.0
+  build_runner_core: ^3.0.0
+  dart_style: '>=1.2.0 <2.0.0'
   glob: ^1.1.0
   logging: ^0.11.0
   package_config: ^1.0.0
   package_resolver: ^1.0.0
-  path: '>=1.2.0 <2.0.0'
-  source_span: ^1.0.0
+  path: '>=1.6.0 <2.0.0'
```

Comment:
Could use ^ syntax here and a few other places.

Reply:
Done.